src/action/input.ts

```diff
@@ -18,6 +19,13 @@ export class Input {
     }))
   }
 
+  get image_url(): string {
```

Comment:
All inputs need to be added to [`action.yml`](https://github.com/cross-platform-actions/action/blob/master/action.yml).

Reply:
done.